ui/src/app/admin/_components/personTable.tsx

```diff
@@ -0,0 +1,284 @@
+                                            : () => void 0
+                                    }
+                                    className={`${header.column.id === 'name' ? 'w-1/4' : 'w-1/12'}`}
+                                >
+                                    <div className="flex items-center">
+                                        {flexRender(header.column.columnDef.header, header.getContext())}
+                                        <SortArrow direction={header.column.getIsSorted()} />
+                                    </div>
+                                </TableHead>
+                            ))}
+                        </TableRow>
+                    ))}
+                </TableHeader>
+                {userInfo === undefined ? (
+                    ''
+                ) : (
+                    <TableBody>
+                        {table.getRowModel().rows.map((row) => (
+                            <TableRow key={row.id}>
+                                {row.getVisibleCells().map((cell) => (
+                                    <TableCell key={cell.id} width={cell.column.columnDef.size}>
+                                        <div className="flex items-center px-2 overflow-hidden whitespace-nowrap">
+                                            <div className={`m-2 ${cell.column.id === 'name' ? 'w-full' : ''}`}>
+                                                {cell.column.id === 'name' && (
+                                                    <div className="flex flex-row">
+                                                        <PersonTableTooltip
+                                                            value={'Manage grouping.'}
+                                                            side={'top'}
+                                                            desc={
+                                                                <Link
+                                                                    href={`/groupings/${cell.row.original.path}`}
+                                                                    rel="noopener noreferrer"
+                                                                    target="_blank"
+                                                                >
+                                                                    <ArrowUpRightFromSquare
+                                                                        size="1.25em"
+                                                                        className="text-text-primary"
+                                                                        data-testid={'arrow-up-right-from-square-icon'}
+                                                                    />
+                                                                </Link>
+                                                            }
+                                                        ></PersonTableTooltip>
```

Comment:
It's preferred to pass in components as children instead of as a prop. Also the props `value` and` side` can be normal strings instead of strings wrapped {}.